src/stories/atoms/typography/heading.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export type HeadingProps = {
+	level: 1 | 2 | 3 | 4;
+	className?: string;
+	children: React.ReactNode;
```

Comment:
```suggestion
	children: ReactNode;
```
No need to import all of React to only use the `ReactNode` type. In fact, importing all of React is a thing of the past nowadays 😉